packer/packer.pkr.hcl

```diff
@@ -0,0 +1,16 @@
+packer {
```

Comment:
I think this is normally `plugins.pkr.hcl`?

Reply:
i'll take your word for it